File: 백준/Silver/1978. 소수 찾기/소수 찾기.cc
#include<stdio.h>

int main(){
    int N;
    scanf("%d",&N);
    int num;
    int cnt=0;
    for(int i=0;i<N;i++){
        scanf("%d",&num);
        if(num==1){
          continue;
        }
        else if(num==2){
          cnt++;
        }
        else{
          for(int j=2;j<num;j++){
              if(num%j==0){
              break;
              }
              if(j==num-1){
                 cnt++;
              }
          }
         }
    }
    printf("%d\n",cnt);
    return 0;
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
n=int(input())
num=input().split()
result=0

for i in range(n):
    x=int(num[i])+1
    cnt=0
    for j in range(1,x):
        if(int(num[i])%j==0):
            cnt+=1
    if(cnt==2):
        result+=1

print(result)
        